docs/waivers.md

```diff
@@ -0,0 +1,54 @@
+# Waivers (Beta)
+
+Waivers is a mechanism to mark controls as "waived" for various reasons
+and to control the running and/or reporting of those controls. It
+takes a YAML input file that identifies which controls are waived,
+(optionally) whether they should be skipped from running, a description of why it
+is waived, and (optionally) an expiration date for the waiver.
+
+NOTE: This mechanism is currently in beta and is ultimately meant to be
+ingested and maintained by Chef Automate UI.
+
+## Usage
+
+To use waivers, you must have a correctly formatted input file and
+invoke `inspec exec` with `--waiver-file [path]`. Currentlty, waiver results are
+only reported with the JSON reporter type, so you must also have
+`--reporter json`. This may change in the future.
+
+```
+% inspec exec --waiver-file waivers.yaml --reporter json -- path/to/profile
```

Comment:
While this is more robust, InSpec docs more frequently give invocations in the order:
`inspec exec PROFILE OPTIONS` and omit the end-of-options marker `--`. It can be visually confusing (is there a space between `-- path/to/profile`)?

Reply:
the end of options marker was there because of the way that thor consumes args when a flag is labelled as an array. I hate it. But I'm removing all mention of json and reporters so that can go away as well.